data/utils/distances_nyx.json

```diff
@@ -0,0 +1,22 @@
+{
+    "nyx_12": 0.8487790952438171,
+    "nyx_11": 0.4781868719689743,
+    "nyx_9": 0.7354352133536897,
+    "nyx_0": 0.19976134544420088,
+    "nyx_3": 0.9120391586424533,
+    "nyx_central": 0.0,
+    "nyx_13": 0.8468468989431717,
+    "nyx_1": 0.8741442119422639,
+    "nyx_7": 1.2193690293154473,
+    "nyx_16": NaN,
```

Comment:
There are some NaN, did you notice this?

Reply:
yes, this is because there are no snapshots without rescalings